### DISCUSSION TITLE
Recuperar Id de solicitud

### DISCUSSION THREAD

**Discussion by TotalConta** [SAT-CFDI/python-satcfdi#14] [Q&A]
Lei la documentacion del SAT y veo que no es posible pero tal vez no la entendi o pase algo por alto ¿es posible recuperar los id de solicitud? me refiero a este id_solicitud='365e2a3d-6f99-4563-856e-28caddc7ad39', para la descarga por sat ws

**Comment by Atjoms** [comment]
No es posible
